### PR TITLE
London Class7-Abdulaziz Alhasan-JavaScript-Core1-WEEK1

### PR DESCRIPTION
**Pull request by AbdulazizAlhasan**
# Your Details

- Your Name:Abdulaziz Alhasan
- Your City:London
- Your Slack Name:Abdulaziz Alhasan

# Homework Details

- Module: JavaScript-Core1
- Week: 1

### CODE REVIEW

File: exercises/F-strings-methods/exercise.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
+let myName="Abdulaziz";
+let length=myName.length;
+let intro= "My name is "+myName+" and my name is "+length+" characters long"
+
+
+
+console.log(intro);
```
Comment (Domenico-cyf) [inline exercises/F-strings-methods/exercise.js:8]: Hi Abdulaziz, well done to your exercise. Just a quick one, perhaps there is too much gap between your lines.

File: mandatory/2-logic-error.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 function getWordLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  
+  return a * b * c;;
```
Comment (soraiya) [inline mandatory/2-logic-error.js:13]: extra colon, small syntax error

File: exercises/F-strings-methods/exercise.js (hunk 1 above)
Comment (AbdulazizAlhasan) [reply exercises/F-strings-methods/exercise.js:8]: Thank you Domenico

File: mandatory/2-logic-error.js (hunk 2 above)
Comment (AbdulazizAlhasan) [reply mandatory/2-logic-error.js:13]: Thank you Soraiya

**Review by soraiya — COMMENTED** [review]
Noticed some spacing between some function parameters and other parts of code missing - separation is better for syntax.  Other than that code is written clearly.

File: exercises/J-functions/exercise2.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 function triple(number) {
   // complete function here
+  return number *=3
```
Comment (akbasmustafa) [inline exercises/J-functions/exercise2.js:3]: you don't need extra assignment operator.  "return (number*3);" is enough for best practice. In addition, You can use formatter and leave space between operator  to make code more readable. The others are great, nice coding.

File: exercises/E-strings-concatenation/exercise.js [hunk 4]
```diff
@@ -1,3 +1,10 @@
 // Start by creating a variable `message`
 
+let message="Hello, my name is "
+let myName="Abdulaziz"
+
+let greeting=message+myName;
+message +=myName;
+
 console.log(message);
```
Comment (mrehmm001) [inline exercises/E-strings-concatenation/exercise.js:9]: Hey Abdulaziz, this looks good, I see you're experimenting both ways to print a message, this is another method:

            var message="Hello, my name is ";
            var name="Abdulaziz";
            console.log(message+name);

You can do the string concatenation inside the console log to achieve the same thing, but needless to say, well done 😁.

**Pull request closed without merging**